source/guide_seafile.rst

```diff
@@ -0,0 +1,271 @@
+.. highlight:: console
+
+.. author:: Finn <mail@f1nn.eu>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/seafile.png
+      :align: center
+
+##########
+Seafile
+##########
+
+Seafile is an enterprise file hosting platform with high reliability and performance. Put files on your own server. Sync and share files across different devices, or access all the files as a virtual disk.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Python_ and its package manager pip
+  * MySQL_
+  * domains_
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://www.seafile.com/en/product/private_server/
+
+Prerequisites
+=============
+
+You'll need your MySQL credentials_. Get them with ``my_print_defaults``:
+
+::
+
+ [isabell@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=isabell
+ --password=MySuperSecretPassword
+ [isabell@stardust ~]$
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+
+Download the seafile server and extract it to a folder called seafile.
+
+::
+
+ [isabell@stardust ~]$ wget https://download.seadrive.org/seafile-server_6.3.4_x86-64.tar.gz
```

Comment:
this can be simplified as
```
mkdir seafile
cd seafile
curl https://download.seadrive.org/seafile-server_6.3.4_x86-64.tar.gz | tar xzf -
mkdir installed
mv seafile-server_* installed/
```
I guess the seafile/installed structure is to enable easy updating? But there don't seem to be any notes on updating in this guide.